opslevel/resource_opslevel_integration_azure_resources.go

```diff
@@ -0,0 +1,223 @@
+package opslevel
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/opslevel/opslevel-go/v2024"
+)
+
+var _ resource.ResourceWithConfigure = &IntegrationAzureResourcesResource{}
+
+var _ resource.ResourceWithImportState = &IntegrationAzureResourcesResource{}
+
+func NewIntegrationAzureResourcesResource() resource.Resource {
+	return &IntegrationAzureResourcesResource{}
+}
+
+type IntegrationAzureResourcesResource struct {
+	CommonResourceClient
+}
+
+type IntegrationAzureResourcesResourceModel struct {
+	Aliases        types.List   `tfsdk:"aliases"`
+	ClientId       types.String `tfsdk:"client_id"`
+	ClientSecret   types.String `tfsdk:"client_secret"`
+	Id             types.String `tfsdk:"id"`
+	LastSyncedAt   types.String `tfsdk:"last_synced_at"`
+	Name           types.String `tfsdk:"name"`
+	SubscriptionId types.String `tfsdk:"subscription_id"`
+	TenantId       types.String `tfsdk:"tenant_id"`
+}
+
+func NewIntegrationAzureResourcesResourceModel(ctx context.Context, planModel IntegrationAzureResourcesResourceModel, azureResourcesIntegration opslevel.Integration) (IntegrationAzureResourcesResourceModel, diag.Diagnostics) {
+	resourceModel := IntegrationAzureResourcesResourceModel{
+		ClientId:       planModel.ClientId,
+		ClientSecret:   planModel.ClientSecret,
+		Id:             ComputedStringValue(string(azureResourcesIntegration.Id)),
+		Name:           RequiredStringValue(azureResourcesIntegration.Name),
+		SubscriptionId: RequiredStringValue(azureResourcesIntegration.SubscriptionId),
+		TenantId:       RequiredStringValue(azureResourcesIntegration.TenantId),
+	}
+
+	if azureResourcesIntegration.LastSyncedAt == nil {
+		resourceModel.LastSyncedAt = types.StringNull()
+	} else {
+		resourceModel.LastSyncedAt = types.StringValue(azureResourcesIntegration.LastSyncedAt.String())
+	}
+	var diags diag.Diagnostics
+	resourceModel.Aliases, diags = types.ListValueFrom(ctx, types.StringType, azureResourcesIntegration.Aliases)
+
+	return resourceModel, diags
+}
+
+func (r *IntegrationAzureResourcesResource) Metadata(ctx context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_integration_azure_resources"
```

Comment:
I think this should be 

```suggestion
	resp.TypeName = req.ProviderTypeName + "_integration_azure"
```

Reply:
@rocktavious the name of the integration is the "Azure Resource Integration" whereas for the aws integration it's "AWS Intgration". This is to differentiate between the "Azure Devops Integration" and "Azure Resources Integration" which target 2 different products.